src/site/xdoc/server/config-listeners.xml

```diff
@@ -25,7 +25,7 @@
 
     <body>
 
-        <section name="Mailbox listeners Configuration">
+        <section name="Mailbox listeners configuration in general">
```

Comment:
What do you mean by "in general" ? overview ?

Reply:
yes!

Reply:
Ok !  It seems a little odd but maybe it's just me.